stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -960,6 +985,10 @@ public class MessageListViewModel(
             }
         }
 
+        if (scrollToMessage?.id == messageId) {
```

Comment:
Do we have to introduce another property for `scrollToMessage`? Can we not just use the same property for focusing a message?